### PR TITLE
Reduce Travis builds

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

**Comment by rkistner** [comment]
What is this change for? It's useful to see the build result on all pull requests.

File: .travis.yml [hunk 1]
```diff
@@ -15,17 +15,22 @@ android:
     - 'android-sdk-preview-license-.+'
     - 'android-sdk-license-.+'
     - 'google-gdk-license-.+'
+
+branches:
+  only:
+    - master
```
Comment (hannesa2) [inline .travis.yml]: This is the only relevant change, the others are just code format

**Comment by hannesa2** [comment]
With this pull request Travis builds just once (and after merge as well) and you see here only one line, otherwise you will see two lines
image

**Comment by rkistner** [comment]
Thanks, makes sense now.

**Pull request merged**